### DISCUSSION TITLE
NeXT release of ravynOS ?

### DISCUSSION THREAD

**Discussion by theokeist** [ravynsoft/ravynos#419] [General]
when can we expect next releade of ravynOS? Also could releaes be periodicaly ie. every 2 weeks or once a month 🙂🙂‍↕️

**Comment by mszoek** [comment]
I'm working on it. There are [automated builds](http://mirrors.nomadlogic.org/nightly/) every time I commit work. They're broken right now which is what I've been focused on lately. Full releases happen when there's a major milestone, so "when" is not very predictable.
